test/onnx/onnx_import.in.cpp

```diff
@@ -1904,3 +1904,59 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, model_reciprocal)
 
     test_case.run();
 }
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_constant_of_shape_float_zeros)
```

Comment:
I don't see that as a big problem, but we have a separate file for dynamic shapes related unit tests, but you can consider moving these 2 tests to `onnx_import_dyn_shapes.in.cpp`.